tests/integration/api_v2/rpc/delegates.test.js

```diff
@@ -0,0 +1,177 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2019 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const config = require('../../../config');
+const { request } = require('../../../helpers/socketIoRpcRequest');
+
+const {
+	resultEnvelopeSchema,
+	jsonRpcEnvelopeSchema,
+	metaSchema,
+} = require('../../../schemas/rpcGenerics.schema');
+
+const {
+	accountSchemaVersion5,
+	dpos,
+} = require('../../../schemas/account.schema');
+
+const wsRpcUrl = `${config.SERVICE_ENDPOINT}/rpc-v2`;
+const getDelegates = async params => request(wsRpcUrl, 'get.accounts', params);
+
+describe('Method get.delegates', () => {
+	let refDelegate;
+	beforeAll(async () => {
+		const response = await getDelegates({ isDelegate: true, limit: 1 });
+		[refDelegate] = response.result.data;
+	});
+
+	describe('returns delegates lists', () => {
+		it('returns top 10 delegates by default', async () => {
+			const response = await getDelegates({ isDelegate: true });
+			expect(response).toMap(jsonRpcEnvelopeSchema);
+			const { result } = response;
+			expect(result).toMap(resultEnvelopeSchema);
+			// expect(result.data).toBeArrayOfSize(10);
+			result.data.forEach(account => {
+				expect(account).toMap(accountSchemaVersion5);
+				expect(account.dpos).toMap(dpos);
+			});
+			expect(result.meta).toMap(metaSchema);
+		});
+
+		// TODO Implement support for search in accounts
+		xit('returns delegates matching search param', async () => {
```

Comment:
Please refer to my earlier comments.
```suggestion
		it('returns delegates matching search param', async () => {
```

Reply:
I will add support for delegate search by search params with issue #261 (SDKv5) and enable tests.